chapter-RDD/RDD.Rmd

```diff
@@ -49,149 +60,996 @@ layout: true
+  
+--
+  
+* In the US, a new born baby weighing less than 1,500 grams is considered to be of "very low birth weight" and receive additional treatment [(Almond et al., 2010)](https://academic.oup.com/qje/article/125/2/591/1882183);
+  
+--
+
+* Flagship state universities use a certain SAT cutoff level to select their students [(Hoekstra, 2009)](https://cdn.theatlantic.com/static/mt/assets/business/Hoekstra_Flagship.pdf);
+
+--
+
+* In Italy, there are quotas of residence permits for illegal immigrants that are allocated on a first-come first-served basis until quota is exhausted [(Pinotti, 2017)](https://pubs.aeaweb.org/doi/pdfplus/10.1257/aer.20150355);
+
+--
+
+We will focus our analysis on the following discontinuity:
+
+* In the US, the legal drinking age is 21 years old [(Carpenter and Dobkin, 2009)](http://masteringmetrics.com/wp-content/uploads/2015/01/Carpenter-and-Dobkin-2009.pdf).
+
+---
+
+# An Example: Alcohol Consumption and Mortality
+
+--
+
+* Imagine you are interested in assessing the __causal__ impact of alcohol consumption by young adults on mortality.
+
+--
+
+* Why is this not that straightforward? Why can't you just regress alcohol consumption on dying age and cause of death?
+
+--
+
+  * Because there may be unobserved selection into alcohol consumption that may also be a determinant of mortality.
+  
+--
+
+* In the US, alcohol consumption is prohibited before the age of 21.
+
+--
+
+* Debate on whether the minimum legal drinking age (MLDA) should be lowered to 18, as was the case in the Vietnam-era.
+
+---
+
+# Key Terms and Intuition
+
+> ***Running variable:*** variable that determines assignment to treatment.
+
+--
+
+$\rightarrow$ $a$ = age
+
+--
+
+> ***Cutoff level:*** level of the ***running variable*** above (or below) which individuals are treated (or not).
+
+--
+
+$\rightarrow$ $c = 21$ year old birthday
 
-* We looked at [Clicking on Heaven's Door](https://www.aeaweb.org/articles?id=10.1257/aer.20150355), which uses the (unknown) time an immigrant quota for a certain employer group is full as a **cutoff** value.
+--
+
+Causal intuition:
+
+* How different are individuals *just before* and *just after* their 21st birthday, other than legal access to alcohol?
+
+--
+
+* Around the threshold, allocation to treatment is ***as good as random***.
+
+--
+
+* `r emo::ji("point_right")` ***Regression discontinuity design*** exploits this allocation to treatment!
+
+---
+
+# Carpenter and Dobkin's data
 
-* Those who are on the immigration website just before and after $c$ are close to identical - except that the early birds got the permit.
+* Let's take a closer at the data used in the paper
+
+.pull-left[
+```{r, echo = TRUE, eval = TRUE}
+# install package containing data
+devtools::install_github("jrnold/masteringmetrics",
+                         subdir = "masteringmetrics")
+
+# load package
+library(masteringmetrics)
```

Comment:
if you don't need the package code, you dont need that call. the `data` call below will work regardless.